### ISSUE TITLE
CREATE/DROP EXTENSION is broken

### ISSUE THREAD

**Issue by soumyadeep2007** [greenplum-db/gpdb-postgres-merge#50]
w/ @melanieplageman 
```
postgres=# create extension gp_inject_fault;
ERROR:  extension "gp_inject_fault" already exists  (seg0 127.0.1.1:25435 pid=14567)
postgres=# drop extension gp_inject_fault;
ERROR:  extension "gp_inject_fault" does not exist 
```

However, as pg_regress does, `CREATE EXTENSION IF NOT EXISTS` works.

**Comment by ashwinstar** [comment]
Seems the extension exists on seg0 and is missing from other segments causing this weird errors for create and drop. So, source of when the extension was left behind on seg0 will give clue to this problem.

**Comment by soumyadeep2007** [comment]
Fixed : c4e16626e5
